### DISCUSSION TITLE
Suggestion: prefix for specific webpage

### DISCUSSION THREAD

**Discussion by shiny-ribbon** [proginosko/LeechBlockNG#455] [Ideas]
As of right now, if you put in a domain name, it will block the entire domain. But wouldn't a prefix for only the homepage be good? (or any specific page). It must be possible to do using regex, but that method is much less intuitive

**Comment by proginosko** [comment]
You can usually do it like this:

`site.com`
`+site.com/*+`

**Comment by shiny-ribbon** [comment]
Oh, that's a clever workaround. Thanks for the help

**Comment by nahoj** [comment]
People reading this might want to understand that the `/` is mandatory to filter on the query path. E.g.

    somesite.com*somepage.html

won't work, but

    somesite.com/*somepage.html

will.

**Reply by proginosko** [reply]
Correct. The single wildcard `*` matches any URL character _except_ forward slash `/`. To match any character _including_ forward slash, use the double wildcard `**`.
